apps/video/js/video.js

```diff
@@ -241,11 +241,14 @@ function initPlayerControls() {
   dom.player.addEventListener('ended', playerEnded);
 
   // handle user tapping events
-  dom.videoControls.addEventListener('click', toggleVideoControls, true);
   dom.play.addEventListener('click', handlePlayButtonClick);
   dom.playerHeader.addEventListener('action', handleCloseButtonClick);
   dom.pickerDone.addEventListener('click', postPickResult);
   dom.options.addEventListener('click', showOptionsView);
+  dom.videoContainer.addEventListener('click', toggleVideoControls);
```

Comment:
What is the reason for removing the videoControls element and instead using the video-container element to toggle the video controls?